python_modules/dagster/dagster/_core/definitions/composition.py

```diff
@@ -937,7 +937,7 @@ def do_composition(
         fn_name=graph_name,
         compute_fn=compute_fn,
         explicit_input_defs=provided_input_defs,
-        exclude_nothing=False,
```

Comment:
One issue is that `do_composition` will expect every `in` to map to a parameter of the graph's compute function unless `exclude_nothing=True`. I did not explore the consequences of flipping this value, but since this code path is used by op-graphs and op-jobs, I suspect there could be many.